app/Repositories/H5pLibrary/H5pLibraryRepositoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Repositories\H5pLibrary;
+
+use App\Models\H5pLibrary;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+interface H5pLibraryRepositoryInterface
```

Comment:
It should extend EloquentRepositoryInterface.

Reply:
Added extend for EloquentRepositoryInterface.